### ISSUE TITLE
Difficulties installing on Mac

### ISSUE THREAD

**Issue by matticervin** [donaldRwilliams/BGGM#26]
I have difficulties installing BGGM on my Mac. 

```
Downloading GitHub repo donaldRwilliams/BGGM@master
✓  checking for file ‘/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmppJE3rn/remotes2695fd22b37/donaldRwilliams-BGGM-38446f5/DESCRIPTION’ ...
─  preparing ‘BGGM’: (478ms)
✓  checking DESCRIPTION meta-information ...
─  cleaning src
─  running ‘cleanup’
─  installing the package to process help pages (395ms)
         -----------------------------------
─  installing *source* package ‘BGGM’ ...
   ** using staged installation
   checking whether the C++ compiler works... yes
   checking for C++ compiler default output file name... a.out
   checking for suffix of executables... 
   checking whether we are cross compiling... no
   checking for suffix of object files... o
   checking whether we are using the GNU C++ compiler... yes
   checking whether clang++ -std=gnu++11 accepts -g... yes
   checking how to run the C++ preprocessor... clang++ -std=gnu++11 -E
   checking for macOS... found
   configure: WARNING: OpenMP unavailable and turned off.
   configure: creating ./config.status
   config.status: creating src/Makevars
   ** libs
   clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/Rcpp/include" -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RcppArmadillo/include" -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RcppDist/include" -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RcppProgress/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include -DARMA_DONT_USE_OPENMP -I../inst/include -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
   clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/Rcpp/include" -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RcppArmadillo/include" -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RcppDist/include" -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RcppProgress/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include -DARMA_DONT_USE_OPENMP -I../inst/include -fPIC  -Wall -g -O2  -c bggm_fast.cpp -o bggm_fast.o
   clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/lib -o BGGM.so RcppExports.o bggm_fast.o -DARMA_DONT_USE_OPENMP -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
   ld: warning: directory not found for option '-L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0'
   installing to /private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs
   ** R
   ** data
   *** moving datasets to lazyload DB
   ** inst
   ** byte-compile and prepare package for lazy loading
   ** help
   Loading required namespace: BGGM
   Registered S3 methods overwritten by 'BFpack':
     method               from
     get_estimates.lm     bain
     get_estimates.t_test bain
   Registered S3 method overwritten by 'GGally':
     method from   
     +.gg   ggplot2
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   Loading required namespace: BGGM
   Failed with error:  ‘unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib’
   *** installing help indices
   *** copying figures
   ** building package indices
   ** installing vignettes
   ** testing if installed package can be loaded from temporary location
   Error: package or namespace load failed for ‘BGGM’ in dyn.load(file, DLLpath = DLLpath, ...):
    unable to load shared object '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so':
     dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
     Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
     Expected in: /usr/lib/libSystem.B.dylib
    in /usr/local/gfortran/lib/libgfortran.5.dylib
   Error: loading failed
   Execution halted
   ERROR: loading failed
─  removing ‘/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/BGGM’
         -----------------------------------
   ERROR: package installation failed
Error: Failed to install 'BGGM' from GitHub:
  System command 'R' failed, exit status: 1, stdout + stderr (last 10 lines):
E>   dlopen(/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/00LOCK-BGGM/00new/BGGM/libs/BGGM.so, 6): Symbol not found: _clock_gettime
E>   Referenced from: /usr/local/gfortran/lib/libgfortran.5.dylib
E>   Expected in: /usr/lib/libSystem.B.dylib
E>  in /usr/local/gfortran/lib/libgfortran.5.dylib
E> Error: loading failed
E> Execution halted
E> ERROR: loading failed
E> * removing ‘/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpEvRDek/Rinst2a57f2e8f2a/BGGM’
E>       -----------------------------------
E> ERROR: package installation failed

```

**Comment by donaldRwilliams** [comment]
Note this was the second error, following an error for the fortran compiler not being found. See here

```
ERROR: package installation failed
Error: Failed to install 'BGGM' from GitHub:
  System command 'R' failed, exit status: 1, stdout + stderr (last 10 lines):
E> clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/lib -o BGGM.so RcppExports.o bggm_fast.o -DARMA_DONT_USE_OPENMP -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
E> ld: warning: directory not found for option '-L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0'
E> ld: warning: directory not found for option '-L/usr/local/gfortran/lib'
E> ld: library not found for -lgfortran
E> clang: error: linker command failed with exit code 1 (use -v to see invocation)
E
```

**Comment by donaldRwilliams** [comment]
The first attempt to fix was to install the fortran compiler here

```
https://github.com/fxcoudert/gfortran-for-macOS/releases/tag/8.2?fbclid=IwAR2SyHWB2BzFcY7bpHYW8VzNvlDsy4Gw1QxUMueXB3H0fXicCWoMbE7Ypac
```

which resulted in the first reported error. The next step was to install `Xcode`

**Comment by matticervin** [comment]
I believe that Xcode is now installed but I still can't install. Here is the error code:

> remotes::install_github("donaldRwilliams/BGGM")
Downloading GitHub repo donaldRwilliams/BGGM@master
tar: Failed to set default locale
tar: Failed to set default locale
   During startup - Warning messages:
   1: Setting LC_CTYPE failed, using "C" 
   2: Setting LC_TIME failed, using "C" 
   3: Setting LC_MESSAGES failed, using "C" 
   4: Setting LC_MONETARY failed, using "C" 
v  checking for file '/private/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T/RtmpzAKfLI/remotes134f1702c4aa/donaldRwilliams-BGGM-71694d9/DESCRIPTION' ...
-  preparing 'BGGM': (472ms)
v  checking DESCRIPTION meta-information ...
-  cleaning src
-  running 'cleanup'
-  installing the package to process help pages (368ms)
   Loading required namespace: BGGM
   Registered S3 methods overwritten by 'BFpack':
     method               from
     get_estimates.lm     bain
     get_estimates.t_test bain
   Registered S3 method overwritten by 'GGally':
     method from   
     +.gg   ggplot2
-  saving partial Rd database (6.5s)
-  cleaning src
-  running 'cleanup'
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'BGGM_2.0.0.tar.gz'
   
Error: (converted from warning) Setting LC_CTYPE failed, using "C"
Execution halted
Error: Failed to install 'BGGM' from GitHub:
  (converted from warning) installation of package '/var/folders/x0/0y_8pyqd0zz1z0hq53cy4nbr0000gn/T//RtmpzAKfLI/file134f534c83d8/BGGM_2.0.0.tar.gz' had non-zero exit status

**Comment by matticervin** [comment]
Now it works!

These are the steps I took:

1. I installed "gfortran-8.2-Mojave.dmg" from here: https://github.com/fxcoudert/gfortran-for-macOS/releases/tag/8.2?fbclid=IwAR2fEwAZx3MgCn5SbocXfyzGhVZ9ipd34widivc-9hqPaDyLvW9bv6KApGM

2. I downloaded "command line tools" following instructions here:
https://www.cnet.com/how-to/install-command-line-developer-tools-in-os-x/?fbclid=IwAR311MTa6E11qoWX5XTwxAD-k5WSJ8BQixBmCKY51qn2CyQojLg_8oenadY

3. I updated to R version 4.0.0 and RStudio version 1.3.959
